ci_framework/roles/libvirt_manager/tasks/attach_interface.yml

```diff
@@ -0,0 +1,35 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# This task requires
+#   vm_name         Domain name to which the interface needs to be attached.
+#   interface_name  source of the device
+#   interface_type  bridge or network
+
+- name: Attach interface to the virtual machine.
+  become: true
```

Comment:
I don't think we need "become: true" if we're using the `-c qemu:///system`

Reply:
Thanks for the info, I wasn't aware. Yes, `become: true` isn't required. Made the required changes and have tested them.